scripts/ci/tags.yaml

```diff
@@ -612,3 +611,49 @@ sample_edge_impulse:
     - nrf/lib/wave_gen/
     - nrf/samples/edge_impulse/
     - zephyr/soc/nordic/
+
+ci_applications_nrf_desktop:
+  files:
+    - bootloader/mcuboot/
+    - modules/crypto/mbedtls/
+    - modules/crypto/oberon-psa-crypto/
+    - modules/crypto/tinycrypt/
+    - modules/lib/open-amp/
```

Comment:
Should we also add `modules/hal/nordic`?

Reply:
Hm, of course this app depends on it (like probably all in nordic), however, I see it like this: nrf_desktop should not verify correctness of nrfx drivers. That should be done by drivers tests. Thus prefer not to add this.

Reply:
ok